tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java

```diff
@@ -57,178 +44,102 @@
 @PrepareForTest({AlluxioMaster.class, BlockMaster.class, Configuration.class, MasterContext.class,
     MetricRegistry.class, UnderFileSystem.class})
 public final class AlluxioMasterRestApiTest extends RestApiTest {
-  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
-  private static final String NOT_ALLUXIO_CONF_PREFIX = "_alluxio_";
-  private AlluxioMaster mAlluxioMaster;
-  private BlockMaster mBlockMaster;
 
   @Before
   public void before() {
-    mAlluxioMaster = PowerMockito.spy(mResource.get().getMaster().getInternalMaster());
-    Whitebox.setInternalState(AlluxioMaster.class, "sAlluxioMaster", mAlluxioMaster);
-    mBlockMaster = PowerMockito.mock(BlockMaster.class);
-    Whitebox.setInternalState(mAlluxioMaster, "mBlockMaster", mBlockMaster);
     mHostname = mResource.get().getHostname();
     mPort = mResource.get().getMaster().getWebLocalPort();
     mServicePrefix = AlluxioMasterRestServiceHandler.SERVICE_PREFIX;
   }
 
   @Test
   public void getCapacityBytesTest() throws Exception {
-    Random random = new Random();
-    long capacityBytes = random.nextLong();
-    Mockito.doReturn(capacityBytes).when(mBlockMaster).getCapacityBytes();
-
+    long memorySize = Configuration.getBytes(Constants.WORKER_MEMORY_SIZE);
```

Comment:
What you're describing sounds like a unit test, but this test is in the integration tests folder and launches a full test Alluxio cluster. This test is still testing the service API, but in an integration as opposed to unit style. It would be useful to also have a unit test which mocks the backend so that we could tell whether the problem is in the service API vs the backend. I've created a ticket for this: https://alluxio.atlassian.net/browse/ALLUXIO-2160